Trust and compliance: Security section (#104)
pretix/pretix-docs: docs/trust/security/isms.md

# ISO 27001

We have implemented an **information security management system** (ISMS) compliant with ISO 27001.
The scope of the ISMS includes our entire company, and therefore specifically both the operation of our SaaS offer and our development processes for pretix itself.

Our compliance with the ISO 27001 standard is validated through yearly internal and external audits.
You can access our current certificate through the [security page](https://pretix.eu/about/en/security) on our website.

> Should you require any additional documentation or have any further questions on the topic, please reach out to us at [support@pretix.eu](mailto:support@pretix.eu).

Reviewer: ```suggestion
If you require additional documentation or have further questions on the topic, reach out to us at [support@pretix.eu](mailto:support@pretix.eu).
```
